electron/src/main.ts

```diff
@@ -104,6 +104,14 @@ if (argv.version) {
 if (argv[ARG.PROXY_ARG]) {
   try {
     proxyInfoArg = new URL(argv[ARG.PROXY_ARG]);
+    if (
+      proxyInfoArg.protocol !== 'http:' &&
```

Comment:
Can this be done: `if (!['http:', 'https:', 'socks4:', 'socks5:'].includes(proxyInfoArg.protocol))`?

Reply:
Or:
```js
if (!/^(https?|socks[45]):/.test(proxyInfoArg.protocol))
```